hostinfo/hostinfo_linux.go

```diff
@@ -48,6 +49,15 @@ func linuxDeviceModel() string {
 	return ""
 }
 
+func getQnapQtsVersion(version_info []byte) string {
```

Comment:
Doesn't follow go naming guidelines.

Reply:
looking into this

Reply:
https://zchee.github.io/golang-wiki/CodeReviewComments/

Reply:
Done. I thought this referred to the function name, just realized it referred to `version_info`

Reply:
Ah yeah, I should have been clearer.